Doc/library/math.rst

```diff
@@ -186,7 +273,10 @@ Number-theoretic and representation functions
 
 .. function:: fmod(x, y)
 
-   Return ``fmod(x, y)``, as defined by the platform C library. Note that the
+   Return the remainder of division ``x / y``,
+   as defined by the platform C library function ``fmod(x, y)``.
+
+   Note that the
```

Comment:
```suggestion
   Return ``fmod(x, y)``, as defined by the platform C library. Note that the
```

This seems to be unrelated to current pr.  Lets just move functions around.

Reply:
I agree, I created issue #126719 to track this modification. I will remove the modification from this PR.